AUTHORS.md

```diff
@@ -0,0 +1,19 @@
+EasyRPG Player authors
+======================
+
+* Alejandro Marzini (vgvgf)
+* Alessio Tosto (Rinnegatamante)
+* Carsten Teibes (carstene1ns)
+* Christian Breitwieser (ChrisBreiti)
+* Diego Pedraza (zegeri)
+* Dmytro Kushnariov (rohkea)
+* Francisco de la Peña (fdelapena)
+* Gabriel Kind (ghabry)
+* Glynn Clements (glynnc)
+* Khelifi Mehdi (BlisterB)
+* Mariano Javier Suligoy (MarianoGNU)
+* Patrick Müssig (Tondorian)
+* Paulo "Zhek" Vizcaino (paulo_v)
+* scurest
```

Comment:
@scurest: Now is the time to tell us how you want to be addressed here... :grinning: